api/api/versions_api.go

```diff
@@ -120,6 +124,7 @@ func (c *VersionsController) CreateVersion(r *http.Request, vars map[string]stri
 		ModelID:     modelID,
 		RunID:       run.Info.RunID,
 		ArtifactURI: run.Info.ArtifactURI,
+		Labels:      versionPost.Labels,
 	}
 
 	version, _ = c.VersionsService.Save(ctx, version, c.MonitoringConfig)
```

Comment:
Can you add test for `Save` operation

Reply:
The test should use a real database instance. You can refer to https://github.com/gojek/merlin/tree/main/api/storage for reference

Reply:
done